src/PresentationalComponents/Inventory/InventoryDetails.js

```diff
@@ -0,0 +1,47 @@
+import {
+  AppInfo,
+  DetailWrapper,
+  InventoryDetailHead,
+} from '@redhat-cloud-services/frontend-components/Inventory';
```

Comment:
As this stuff is closely tied with RHEL Inventory/systems, thinkin yer probably not gonna wanta use it for anything (doesn't have clusters 🤔 )

Reply:
Ah, alright, I thought `inventory` is abstracted from the context of systems/clusters/whatever.